### PR TITLE
Error reporting improvements

### PR DESCRIPTION
**Pull request by andrewrlee**
### Context

https://trello.com/c/UmcLRNoN

### Intent

As below

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -70,7 +70,8 @@ const createApp = services => {
       const transactionName = samplingContext?.transactionContext?.name;
       return transactionName &&
         (transactionName.includes('/health') ||
-          transactionName.includes('/public/'))
+          transactionName.includes('/public/') ||
+          transactionName.includes('/assets/'))
```
Comment (andrewrlee) [inline server/app.js:74]: excluding assets from transactions sent to sentry

File: server/app.js [hunk 2]
```diff
@@ -255,7 +256,13 @@ const createApp = services => {
   app.use(routes(services, establishmentData));
   // the sentry error handler has to be placed between our controllers and our error handler
   // https://docs.sentry.io/platforms/node/guides/express/
-  app.use(Sentry.Handlers.errorHandler());
+  app.use(
+    Sentry.Handlers.errorHandler({
+      shouldHandleError(error) {
+        return !(error instanceof NotFound);
+      },
```
Comment (andrewrlee) [inline server/app.js:263]: do not allow sentry to handle NotFound errors

File: server/repositories/cmsApi.js [hunk 3]
```diff
@@ -53,7 +53,7 @@ class CmsApi {
     try {
       return await call();
     } catch (err) {
-      if (statusCodes.includes(err.response.status)) {
+      if (err.response && statusCodes.includes(err.response.status)) {
```
Comment (andrewrlee) [inline server/repositories/cmsApi.js:56]: Do not blow up if we get a timeout (which has no response)

Comment (jonhindle) [reply server/repositories/cmsApi.js:56]: You could just use optional chaining.
```if (statusCodes.includes(err?.response?.status)) {```

**Review by jonhindle — APPROVED** [review]
fine

File: server/repositories/cmsApi.js (hunk 3 above)
Comment (andrewrlee) [reply server/repositories/cmsApi.js:56]: I did and then undid it, just seemed slightly weird doing a check to see if the array contained undefined.. I'll put it back.

**Pull request merged**